assets/respec.css

```diff
@@ -79,7 +79,7 @@ cite .bibref {
 }
 
 code {
-  color: #c83500;
+  color: #c63501;
```

Comment:
- `#c83500` (or `hsl(15.9, 100%, 39.2%)`) (color contrast: 4.48, fail)
- `#c63501` (or `hsl(15.8, 99%, 39.0%)`) (color contrast: 4.54, pass)
(at least the a11y linter is not labeling 20% of page as error)